docs_user/modules/con_openshift-preparation-for-block-storage-adoption.adoc

```diff
@@ -2,14 +2,12 @@
 
 = {OpenShiftShort} preparation for {block_storage} adoption
 
-Before you deploy {rhos_prev_long} ({OpenStackShort}) in {rhocp_long}, ensure that the networks are ready, that you decide which {OpenShiftShort} nodes to restrict, and that you make any necessary changes to the {OpenShiftShort} nodes.
-
-//kgilliga: Need to change xref:about-node-selector_planning[About node selector] to the link to RHOCP documentation if I remove this module. (https://docs.openshift.com/container-platform/4.16/nodes/scheduling/nodes-scheduler-node-selectors.html)
+Before you deploy {rhos_prev_long} ({OpenStackShort}) in {rhocp_long} nodes, ensure that the networks are ready, that you decide which {OpenShiftShort} nodes to restrict, and that you make any necessary changes to the {OpenShiftShort} nodes.
```

Comment:
This sentence without the attributes translates to "Before you deploy RHOSP in RHOCP nodes". Is that accurate?

Reply:
Yes, you need to consider all those things before you begin the adoption.